Microsoft.Recognizers.Resources/CommonDefinitions.ttinclude

```diff
@@ -10,7 +10,7 @@
 <#@ import namespace="Microsoft.Recognizers.Resources.Common" #>
 <#
 	// Read the model file 
-	var input = new StringReader(File.ReadAllText(this.DataFilename));
+	var input = new StringReader(File.ReadAllText(Host.ResolveAssemblyReference("$(SolutionDir)") + this.DataFilename)); 

```

Comment:
Good catch! We've tested your update in a couple of boxes and worked fine. Thanks!